addons/l10n_it/models/account_move.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _extend_with_attachments(self, attachments, new=False):
+        return super(AccountMove, self.with_context(
+            account_predictive_bills_disable_prediction=True,
+            ))._extend_with_attachments(attachments, new)
```

Comment:
Please close a bracket/parenthesis on the same level as it was opened

```suggestion
        ))._extend_with_attachments(attachments, new)
```